pkg/apis/build/v1alpha1/abstract_builder.go

```diff
@@ -0,0 +1,14 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type BuilderResource interface {
```

Comment:
This file is still named abstract_builder.go

Reply:
🙂